### ISSUE TITLE
JCenter is shutting down

### ISSUE THREAD

**Issue by prfarlow1** [jsibbold/zoomage#104]
https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

It appears this library is uploaded and distributed through JCenter, which is shutting down in 3 months. @jsibbold  will you update this library to publish to MavenCentral or JitPack instead?

**Comment by jsibbold** [comment]
Yes I will try to find it a new home. I will give an update here once I figure it out.

**Comment by jzbrooks** [comment]
MavenCentral would be nice. I've setup a couple of maven packages there. It isn't too bad and the effort is a pay once cost.
https://chris.banes.dev/publishing-to-maven-central/

I'd be happy to help with this, if you'd like @jsibbold

**Comment by jsibbold** [comment]
1.3.1 now on Maven Central
